WebAPI/Program.cs

```diff
@@ -12,8 +11,13 @@
 
 // Add services to the container.
 
+builder.Services.AddCors(options =>
+     options.AddDefaultPolicy(builder =>
+     builder.AllowAnyHeader().AllowAnyMethod().AllowAnyOrigin()));
```

Comment:
I would recommend to put into a column this chain of methods call for better reading, like:
```suggestion
     builder
          .AllowAnyHeader()
          .AllowAnyMethod()
          .AllowAnyOrigin()));
```

Advised this approach before